go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/dancannon/gorethink v4.0.0+incompatible
 	github.com/dapr/kit v0.0.2-0.20210614175626-b9074b64d233
 	github.com/deepmap/oapi-codegen v1.8.1 // indirect
-	github.com/denisenkom/go-mssqldb v0.0.0-20191128021309-1d7a30a10f73
+	github.com/denisenkom/go-mssqldb v0.0.0-20210411162248-d9abbec934ba
```

Comment:
Is it necessary?

Reply:
Yes it absolutely is necessary. Please see the linked bug and discussion in Discord. Today the SQL Server component is broken with Go 1.16 and updating the module to this particular version (this is the commit corresponding to the latest tag) fixes it.

Reply:
https://github.com/denisenkom/go-mssqldb/issues/688